lib/event/domain/notification/verification-request.php

```diff
@@ -16,34 +16,29 @@
  * if not, see https://www.gnu.org/licenses.
  */
 
-namespace Automattic\Domain_Services_Client\Event\Contact\Verification;
+namespace Automattic\Domain_Services_Client\Event\Domain\Notification;
 
 use Automattic\Domain_Services_Client\{Event};
 
 /**
- * Verification request event
+ * Domain verified event
  *
- * This event indicates that a verification request was sent to the contact's email address.
+ * - This event is generated when a verification is requested for the domain's contact owner
+ * - This event contains the email linked to the domain.
+ *     - It can be retrieved with the `get_email` method
+ *
+ * @see \Automattic\Domain_Services_Client\Event\Domain\Notification\Verification_Success
  */
-class Request implements Event\Event_Interface {
+class Verification_Request implements Event\Event_Interface {
 	use Event\Data_Trait;
-	use Event\Object_Type_Contact_Trait;
-
-	/**
-	 * Returns the email address that the verification request was sent to
-	 *
-	 * @return string|null
-	 */
-	public function get_email(): ?string {
-		return $this->get_data_by_key( 'event_data.email' );
-	}
+	use Event\Object_Type_Domain_Trait;
 
 	/**
-	 * Returns the domain associated with the event's contact handle
+	 * Returns the email linked to the domain
 	 *
 	 * @return string
 	 */
-	public function get_associated_domain(): string {
-		return $this->get_data_by_key( 'event_data.associated_domain' ) ?? '';
+	public function get_email(): string {
+		return $this->get_data_by_key( 'event_data.email' );
```

Comment:
Since `event_data.email` can be null on the server, side (`get_verification_code(): ?string`), I guess we should either check for that here and return an empty string or allow this method to return null.

Reply:
Never mind.  @gius80 pointed out to me that we'll never have this value be null in the client since these are filtered out on the server side.